HW_3/Task_20.py

```diff
@@ -25,12 +25,18 @@
 'И': 1, 'Н': 1, 'О': 1, 'Р': 1, 'С': 1, 'Т': 1, 'D': 2, 'G': 2, 'Д': 2, 'К': 2, 'Л': 2, 'М': 2, 'П': 2,
 'У': 2, 'B': 3, 'C': 3, 'M': 3, 'P': 3, 'Б': 3, 'Г': 3, 'Ё': 3, 'Ь': 3, 'Я': 3, 'F': 4, 'H': 4, 'V': 4,
 'W': 4, 'Y': 4, 'Й': 4, 'Ы': 4, 'K': 5, 'Ж': 5, 'З': 5, 'Х': 5, 'Ц': 5, 'Ч': 5, 'J': 8, 'X': 8, 'Ш': 8,
-'Э': 8, 'Ю': 8, 'Q': 10, 'Z': 10, 'Ф': 10, 'Щ': 10, 'Ъ': 10,}
+'Э': 8, 'Ю': 8, 'Q': 10, 'Z': 10, 'Ф': 10, 'Щ': 10, 'Ъ': 10}
 word = str.upper(input('Введите слово: '))
 points = 0
-for i in word:
-    if i in scrable:
-        points += scrable[i]
+# for i in word:
+#     if i in scrable:
+#         points += scrable[i]
+# print(f'{points} - очков')
+if word[0] in scrable:
+ for i in range(len(word)):
+       points += scrable.get(word[i], 0)
 print(f'{points} - очков')
```

Comment:
Зачем вам столько лишнего?
```suggestion
for char in word:
    points += scrable.get(char, 0)
print(f'{points} - очков')
```
вот и все

Reply:
Можно мержить

Reply:
вы классно ведете семинары и дискутировать с вами приятно))
спасибо вам)

Reply:
Не за что. Я мне в радость. Обращайтесь